cockroachdb-in-comparison.md

```diff
@@ -2,3 +2,19 @@
 title: CockroachDB in Comparison
 toc: false
 ---
+
+| | CockroachDB | MySQL | PostgreSQL | Cassandra | HBase | MongoDB | Riak 
+--|--|--|--|--|--|--|--
+Automated Scaling | Yes | No | No | Yes | Yes  | Yes | Yes 
+Automated Failover & Repair | Yes | No | No | Yes | Yes | Yes | Yes 
+Strong Consistency | Yes | N/A | N/A | No | No | No | No   
+Read from any Replica | No | N/A | N/A | Yes | Yes | Yes | Yes  
+Distributed Transactions| Yes | No | No | No | No | No | No 
+ACID Semantics | Yes | Yes | Yes | No | Row-only | Document-only | No   
+SQL| Yes | Yes | Yes | No | No* | No | No 
+Open Source| Yes | Yes | Yes | Yes | Yes | Yes | Yes 
+
+
+Need to find a way to provide addition detail on hover:
+
+\* Apache Pheonix can provide a SQL layer on top 
```

Comment:
Typo: Phoenix

And it's [not the only one](http://www.infoworld.com/article/2683729/hadoop/10-ways-to-query-hadoop-with-sql.html). It's not even the only one under the apache umbrella: [trafodion](http://trafodion.apache.org/). I think it would be best to only consider plain hbase here, and if we want to talk about a sql layer on top of hbase then it should get its own column (since some of the sql layers may have different answers for some of the other questions)